gb-cpu/src/trait.rs

```diff
@@ -0,0 +1,8 @@
+use crate::address_bus::Error;
+
+trait AddressBus {
```

Comment:
You should rename the gb-rom folder to gb-memory or something and move this code there.

I won't comment on the implementation choice or the names, I just need a read and write function and it should work.

Reply:
done